### DISCUSSION TITLE
Migrate from postgresql to ms sql , how to handle large data for more than 1000 chars in ms sql with prisma

### DISCUSSION THREAD

**Discussion by gopeshvlinkinfo** [prisma/prisma#10664] [Q&A] [answered]
i am going to migrate from postgresql to ms sql database. i want to know how t handle large data for more than 1000 chars in ms sql qith prisma ?

**Comment by austincrim** [comment, marked as answer]
Hey @gopeshvlinkinfo 👋 !

The Prisma schema's default `String` type defaults to `nvarchar(1000)` in SQL Server. You can override this using our native database type attribute `@db`.

```prisma
model User {
  id   Int    @id @default(autoincrement())
  name String @db.NVarChar(1500)
}
```

The `name` field with then use `nvarchar(1500)` in the underlying table definition. You can read more about this [in our docs](https://www.prisma.io/docs/concepts/components/prisma-migrate/supported-types-and-db-features#mapping-fields-to-a-specific-native-type).

Let me know if you have any questions!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
